libsolidity/ast/ASTPrinter.cpp

```diff
@@ -78,6 +78,13 @@ bool ASTPrinter::visit(InheritanceSpecifier const& _node)
 	return goDeeper();
 }
 
+bool ASTPrinter::visit(UsingForDirective const& _node)
+{
+	writeLine("UsingDirective");
+	printSourcePart(_node);
```

Comment:
What will this print for a the node? Just `UsingDirective`?

This entire AST printer is not very useful.

Reply:
I added it for completeness. It will correctly indent the sub nodes of `UsingForDirective`.